app/views/movies/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @movie.title %></h1>
+<h4>Creation Year: <%= @movie.creation_year %></h4>
+<h4>Genre: <%= @movie.genre %></h4>
+<div id="movie-<%= @movie.id %>-actors">
+  <% @actors.order_by_age.each do |actor| %>
```

Comment:
This method (and your other class method you made) are fine; personally I would make a method called `actors_ordered_by_age` on a `Movie` model as an instance method, and use the `actors` association in that model instead of making it a class method. That way, you know you're only ever accessing the actors from that movie. And if you needed to order all actors regardless of movie association elsewhere, then you'd make a class method. But as long as it's tested correctly and thoroughly, this way of doing it is fine.